METRICS.md

```diff
@@ -24,7 +24,36 @@ scrape_configs:
       - targets: [ "emr-header-1:9098","emr-worker-1:9096","emr-worker-2:9096","emr-worker-3:9096","emr-worker-4:9096" ]
 ```
 
-3.You need to install Grafana server(https://grafana.com/)
+3.You need to install Grafana server(https://grafana.com/grafana/download)
+4.Import RSS dashboard into grafana.
```

Comment:
Let's give details about how to import RSS dashboard here, including where to find the json